src/AccessibilityInsights.AutomationTests/CommandParametersUnitTests.cs

```diff
@@ -74,6 +76,7 @@ public void Ctor_ConfigFileDoesNotExist_ThrowsAutomationException_ErrorAutomatio
             }
         }
 
+        #if FAKES_SUPPORTED
```

Comment:
It looks in general, the #if indentation is aligned with its method whereas the #endif line has no indentation. It might be nicer for both to be consistent

Reply:
The indent is now uniform, although I can't really tell if it looks great.